src/averageAges.js

```diff
@@ -20,6 +20,30 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let num = 0;
+  const sum = people.reduce(calculateSum, 0);
+
+  function calculateSum(prev, person) {
+    if (century) {
+      if (person.sex === 'm' && Math.ceil(person.died / 100) === century) {
```

Comment:
you duplicate `person.sex === '...'` all over your code, so you can create a separate function allowing you to filter people by their sex, and then you just like
```
const men = ...;
const menDiedInCentury = men.filter(...);
```